### ISSUE TITLE
German README.de.MD doesn't match README.md

### ISSUE THREAD

**Issue by lukasvrenner** [swaywm/sway#7812]
### Please read the following before submitting:
- We are not accepting any new window management features unless they get implemented by i3.

### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.

The German README.de.MD doesn't match the English one. I'm guessing it just needs to be updated, but the German one contains information regarding maintaining distro packages which can isn't there on the English README.md

**Comment by kennylevinsen** [comment]
Patches welcome. :)
